### DISCUSSION TITLE
osu!lazer death after deleting any from screen in skin editor

### DISCUSSION THREAD

**Discussion by Zoleks** [ppy/osu#15295] [Q&A] [answered]
Found a bug when you delete anything on screen in skin editor, osu!lazer just hiding all interface on song selection screen and main menu, but key bindings still working like open settings with Ctrl+O.
Another people have game crashed right after closing skin editor.
P.S Bug working and on default lazer skin
Bug demonstration: https://youtu.be/sDOf0PrzaRU

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15199
